src/pages/multipleProducts.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react';
+import Layout from '../layout/Layout';
+
+const title = 'XXXXXXXX - Fares data build tool';
```

Comment:
Why do we have XXXXXXX here?

Reply:
This is just a placeholder page I've created to check my cookie was actually being created. Checked with Rob for the page name so that his code will overwrite this placeholder page.